example/rlottiePlayer/rlottie.h

```diff
@@ -0,0 +1,525 @@
+/*
```

Comment:
Please don't attach rlottie.h 
rlottie already has it rlottie/inc folder.
this project could use it instead of the copy.